apps/web/components/sections/waitlist/waitlist-modal.tsx

```
@@ -0,0 +1,467 @@
+					return (
+						<div key={label} className="flex items-center">
+							<div
+								className={`h-2 w-8 rounded-full mx-1 ${
+									isActive
+										? 'bg-blue-600'
+										: isCompleted
+											? 'bg-green-600'
+											: 'bg-gray-200'
+								}`}
+								aria-label={`Step ${stepNumber}: ${label}`}
+							/>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)